app/lib/utils/file.js

```diff
@@ -0,0 +1,10 @@
+// Taken from https://github.com/kevva/executable and modified
```

Comment:
Can you write this in the comment and ideally open an issue on `executable` about making it more configurable. Would be nice to be able to just depend on that module instead of copy-paste.